README.md

```diff
@@ -61,26 +71,23 @@ optional arguments:
                         Model types: "dense-unet", "unet" (default: "dense-
                         unet")
   --model_wts_path MODEL_WTS_PATH
-                        Path for model wts to be used (default='./saved_model_
-                        wts/dense_unet_back2front_finetuned.pth')
-  --is_mgz [IS_MGZ]     Is the image in .mgz format (default=False, default
-                        format is .nii.gz)
-  --save_prob [SAVE_PROB]
-                        Should the softmax prob values for each voxel be saved
-                        ? (default: False)
-  --use_gpu [USE_GPU]   Use GPU for inference? (default: True)
+                        Path for model wts to be used, provide a model from
+                        saved_model_wts/
+  --is_mgz              Use this flag when image is in .mgz format
+  --save_prob           Use this flag to save the softmax prob values for each
+                        voxel
+  --use_gpu             Enable GPU usage (CPU is used by default)
```

Comment:
When `--use_gpu` (and other flags) is for just enabling an option, it should be defined as above. See further explanation on `perform_pred.py` below.